terraform/modules/pre-auth-lambda/data.tf

```diff
@@ -0,0 +1,5 @@
+data "archive_file" "lambda_user_expiry_zip" {
+  type        = "zip"
+  source_dir  = "${path.module}/lambda"
+  output_path = "${path.module}/lambda-user-expiry.zip"
```

Comment:
might want to rename this to match the function name

Reply:
This is done.